bin/check-redis-memory-percentage.rb

```diff
@@ -80,8 +80,12 @@ def run
     redis = Redis.new(options)
 
     redis_info = redis.info
-    memory_in_use = redis_info.fetch('used_memory').to_f.div(1024).to_f             # used memory in KB (KiloBytes)
-    total_memory = redis_info.fetch('maxmemory', system_memory).to_f.div(1024).to_f # max memory (if specified) in KB
+    max_memory = redis_info.fetch('maxmemory', 0).to_f
```

Comment:
I am trying to preserve the existing logic, of getting the value from `redis.info`, converting to a float, and then doing the division. I think the reason the original logic was to convert to a float rather than an integer was so we weren't doing integer division. If that's not necessary, then I can change the it to an integer. I can also move the conversion to right before the division to calculate `total_memory` if that will be more clear